atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -91,6 +92,7 @@ private AtlasDbConstants() {
             PARTITION_MAP_TABLE,
             PERSISTED_LOCKS_TABLE,
             SWEEP_PROGRESS_TABLE,
+            COORDINATION_TABLE,
```

Comment:
There's also the `ATOMIC_TABLES` set, although it's missing some other tables too (targeted sweep table ref dictionary).

Reply:
Added to `ATOMIC_TABLES` - I'm not aware if it's particularly significant, but indeed this table would be unhappy on a non-pUE supporting KVS